lib/firehose/long_poll_transport.coffee

```diff
@@ -108,7 +108,7 @@ class LongPollTransport extends Transport
 
     unless @_needToNotifyOfReconnect or @_stopRequestLoop
       @_needToNotifyOfReconnect = true
-      @config.disconnected()
+      @config.disconnected(jqXhr, status, error)
```

Comment:
Ideally, yes. Although as the other events already have some data passed in, it would be hard to define a standard way. Perhaps that change would be better waiting for a rewrite/refactor/v2 to standardize them.